Modules/socketmodule.c

```diff
@@ -7711,6 +7711,26 @@ PyInit__socket(void)
     PyModule_AddIntMacro(m, ALG_OP_VERIFY);
 #endif
 
+/* Ethernet Protocol ID's */
```

Comment:
``grep -R 'define ETH_P_' /usr/include/`` on my system shows about 100 constants. Please add a comment that explains your selection and mentions that there are more constants.

Reply:
I've only added the most popular ones in my opinion, do you think we need to add more? all of them? or just add a comment explaining I've included the most popular ones?

Reply:
What makes them popular? This should be explained in the comment.